django/views/templates/default_urlconf.html

```diff
@@ -337,78 +218,49 @@
+          <path class="exhaust__line" fill="#E6E9EE" d="M250 172h7v90h-7z"/>
+          <path class="flame" d="M250.27 178.834l-5.32-8.93s-2.47-5.7 3.458-6.118h10.26s6.232.266 3.306 6.194l-5.244 8.93s-3.23 4.37-6.46 0v-.076z" fill="#AA2247"/>
+        </svg>
+        <h1>{% translate "The install worked successfully! Congratulations!" %}</h1>
         <p>{% blocktranslate %}You are seeing this page because <a href="https://docs.djangoproject.com/en/{{ version }}/ref/settings/#debug" target="_blank" rel="noopener">DEBUG=True</a> is in your settings file and you have not configured any URLs.{% endblocktranslate %}</p>
       </main>
       <footer class="u-clearfix">
-        <a href="https://docs.djangoproject.com/en/{{ version }}/" target="_blank" rel="noopener">
-        <div class="option one">
-            <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-                <defs>
-                    <path d="M0 0h24v24H0V0z" id="a"></path>
-                </defs>
-                <clipPath id="b">
-                    <use overflow="visible" xlink:href="#a"></use>
-                </clipPath>
-                <path clip-path="url(#b)" d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6C7.8 12.16 7 10.63 7 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z"></path>
-            </svg>
-            <div>
-                <h4>{% translate "Django Documentation" %}</h4>
-                <p>{% translate 'Topics, references, &amp; how-to’s' %}</p>
-            </div>
-        </div>
-      </a>
-      <a href="https://docs.djangoproject.com/en/{{ version }}/intro/tutorial01/" target="_blank" rel="noopener">
-        <div class="option two">
-          <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-            <path d="M0 0h24v24H0V0z" fill="none"></path>
-            <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"></path>
+        <a class="option one" href="https://docs.djangoproject.com/en/{{ version }}/" target="_blank" rel="noopener">
+          <svg viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+            <path d="M9 21c0 .55.45 1 1 1h4c.55 0 1-.45 1-1v-1H9v1zm3-19C8.14 2 5 5.14 5 9c0 2.38 1.19 4.47 3 5.74V17c0 .55.45 1 1 1h6c.55 0 1-.45 1-1v-2.26c1.81-1.27 3-3.36 3-5.74 0-3.86-3.14-7-7-7zm2.85 11.1l-.85.6V16h-4v-2.3l-.85-.6A4.997 4.997 0 017 9c0-2.76 2.24-5 5-5s5 2.24 5 5c0 1.63-.8 3.16-2.15 4.1z"></path>
           </svg>
-          <div>
-            <h4>{% translate "Tutorial: A Polling App" %}</h4>
-            <p>{% translate "Get started with Django" %}</p>
-          </div>
-        </div>
+          <p><span class="option__heading" aria-label="{% translate 'Django Documentation' %}.">{% translate "Django Documentation" %}</span><br>
```

Comment:
This full stop is so VoiceOver on macOS marks a pause when switching from line to line. It’s annoying to have to repeat the whole label just to add a ".", but this felt more appropriate than introducing a "screen-reader only" class int he styles.